code/ai/aiturret.cpp

```diff
@@ -1904,7 +1904,9 @@ bool turret_fire_weapon(int weapon_num, ship_subsys *turret, int parent_objnum,
 							points = 1;
 						}
 
-						if (swp->primary_bank_ammo[bank_to_fire] >= 0) {
+						if (swp->primary_bank_ammo[bank_to_fire] > 0) {
```

Comment:
In this case, we probably want `points`, since that's what it's going to attempt to subtract, since it might be more than one shot it's doing.

Reply:
Good point, pushed. Also altered the logical operators to match.